buildconfig/version.py.in

```diff
@@ -27,3 +27,46 @@ The python version information should always compare greater than any previous
 releases. (hmm, until we get to versions > 10)
 """
 
+class VersionInfo(object):
+    def __init__(self, major, minor, patch):
+        self._version = (major, minor, patch)
+
+    @property
+    def major(self):
+        return self[0]
+
+    @property
+    def minor(self):
+        return self[1]
+
+    @property
+    def patch(self):
+        return self[2]
+
+    def __repr__(self):
```

Comment:
You might want to consider giving different output for `__repr__` based on how the [docs](https://docs.python.org/3/library/functions.html#repr) describe it. Some ideas:

```python
# Use the class
def __repr__(self):
    return '{}({}, {}, {})'.format(self.__class__.__name__, *self._version)

# or use the tuple as a string (like it did originally)
def __repr__(self):
    return repr(self._version)

# or use <>'s (to indicate that it cannot be evaluated) with your current string data
def __repr__(self):
    return repr('<pygame.version.vernum(major={}, minor={}, patch={})>'.format(*self._version))

# or use <>'s with the __str__ output
def __repr__(self): 
    return '<{}>'.format(self.__str__())
 ```

Reply:
sys.version_info doesn't follow that rule, but I guess the first one is fine